content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,452 @@
+title: darktable 3.2: containment effect!
+author: Nilvus
+slug: darktable-3-2
+date: 2020-08-10
+lede: 2013_Island_Kerlingarfjoll.jpg
+lede_author: AlicVB
+tags: announcement, darktable-release
+
+In other languages:
+[Français](https://linuxfr.org/news/darktable-3-2-l-effet-confinement),
+[Italiano](https://www.mauriziopaglia.it/darktable-3-2/),
+[Deutsch](https://www.bilddateien.de/blog/2020-08-10-darktable-3.2-auswirkungen-des-lockdowns.html).
+
+For the first time in its history, darktable breaks its one year release cycle by releasing version 3.2 in August of 2020. This year's containment, already quite peculiar, has led to a marked increase in contributions and improvements. On top of that, version, 3.4 is still scheduled for Christmas 2020. 2020 will therefore be the first year in which the darktable team will have the pleasure to offer you two major versions.
```

Comment:
If translations (FR/IT/DE) use "peculiar containment", there's no problem in EN version using global health and lockdown issues since most of European countries managed to go through lockdown and are in "mostly recovering state". English speaking countries (no pointing fingers) do have different perspective so changing "this year's containment" to "state of global health" doesn't change meaning nor spirit of the paragraph.

Reply:
And so, that part will not change. From my view on Covid, "unfortunate state of global health" is not totally the same.